crates/bevy_ecs/src/system/query.rs

```diff
@@ -634,7 +634,7 @@ impl<'w, 's, D: QueryData, F: QueryFilter> Query<'w, 's, D, F> {
         entities: EntityList,
     ) -> QueryManyIter<'_, 's, D::ReadOnly, F, EntityList::IntoIter> {
         // SAFETY:
-        // - `self.world` has permission to access the required components.
+        // - `self.world` has permission to access the related components.
```

Comment:
Same as above: this doesn't appear to be related to required/related components, I think this is just a find&replace oopsie!